torch/_higher_order_ops/while_loop.py

```diff
@@ -37,13 +37,13 @@ def __call__(
         additional_inputs: tuple[Union[torch.Tensor, torch.SymInt, int], ...],
         /,
     ):
-        if not isinstance(carried_inputs, tuple):
+        if not isinstance(carried_inputs, (tuple, list)):
```

Comment:
this is bad.  after serde, the tuple inputs are turned in list inputs. Filed an issue for tracking.